embed-code-go/embed_code.go

```diff
@@ -38,7 +39,8 @@ const (
 // Embedding is the process that consists of the following steps:
 //   - the code fragments are extracted from the code files;
 //   - the docs files are scanned for <embed-code> tags;
-//   - for each tag, the code fragments are embedded into the docs. The embedding is parametrized with the tag attributes.
+//   - for each tag, the code fragments are embedded into the docs. The embedding is parametrized with
+//     the tag attributes.
```

Comment:
This seems to be related to the line wrap setting in the IDE configuration. Let's commit the IDE settings as well, if they aren't already included. This ensures that everyone working with the code will have the correct line length settings.

Reply:
The length is being checked with `lll` linter, and the default maximum size is set to 120 symbols.

See: https://github.com/SpineEventEngine/embed-code/actions/runs/10705034029/job/29679432200?pr=44#step:6:29

Reply:
@olena-zmiiova can we make it 100?

Reply:
Yes, I updated it within this PR: https://github.com/SpineEventEngine/embed-code/pull/44